tests/testthat/test-line_length_linter.R

```diff
@@ -30,4 +29,11 @@ test_that("returns the correct linting", {
   expect_lint("aaaaaaaaaaaaaaaaaaaab",
     rex("Lines should not be more than 20 characters"),
     line_length_linter(20))
+
+  # Don't duplicate lints
+  expect_length(
+    lint("x <- 2 # ------------\n",
```

Comment:
In tests, calls to 'lint' should have `parse_settings = FALSE` shouldn't they?

Reply:
absolutely!